File: app/src/main/java/shts/jp/android/nogifeed/NogiFeedApplication.java
package shts.jp.android.nogifeed;

import android.app.Application;
import android.content.Intent;

import shts.jp.android.nogifeed.receivers.TokenRegistrationService;

public class NogiFeedApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        startService(new Intent(this, TokenRegistrationService.class));
    }
}

File: app/src/main/java/shts/jp/android/nogifeed/NogiFeedApplication.kt
package shts.jp.android.nogifeed

import android.app.Application
import android.content.Intent
import com.facebook.stetho.Stetho

import shts.jp.android.nogifeed.receivers.TokenRegistrationService

class NogiFeedApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        Stetho.initializeWithDefaults(this)
        startService(Intent(this, TokenRegistrationService::class.java))
    }
}
